scrapy/core/scheduler.py

```diff
@@ -107,7 +108,23 @@ def _dqpush(self, request):
             return
         try:
             self.dqs.push(request)
-        except ValueError as e:  # non serializable request
+        except TransientError as e:
+            msg = "Unable to push request to queue: %s"
```

Comment:
Hm, does `%s` work here? Which argument does it pick up to do the string formatting?

Reply:
It picks exception message, e.g. for https://github.com/whalebot-helmsman/scrapy/blob/dbf2711e39876bad1077e49a38106d9473977864/tests/test_scheduler.py#L420 message will be something like
```
WARNING  scrapy.core.scheduler:scheduler.py:284 Unable to push request to queue: This class raises an Tr
ansientError                                                                                            
Traceback (most recent call last):                                                                      
  File "/home/nikita/dev/scrapy/scrapy/core/scheduler.py", line 281, in _dqpush                         
    self.dqs.push(request)                                                                              
  File "/home/nikita/dev/scrapy/scrapy/pqueues.py", line 80, in push                                    
    q.push(request)  # this may fail (eg. serialization error)                                          
  File "/home/nikita/dev/scrapy/tests/test_scheduler.py", line 378, in push                             
    raise TransientError('This class raises an TransientError')                                         
scrapy.exceptions.TransientError: This class raises an TransientError 
```

Reply:
TIL :)